src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.tsx

```diff
@@ -83,6 +87,7 @@ export const PrivateChats: React.FC<PrivateChatsProps> = ({
 
   const numberOfSearchResults = renderedSearchResults.length;
   const hasChatPreviews = renderedPrivateChatPreviews.length > 0;
+  const numberOfOnline = renderedOnlineUsers.length;
```

Comment:
`numberOfOnlineUsers`

Reply:
And please, change the text of the label as suggested by @0xdevalias to `${numberOfTherOnlineUsers} other online users`